app/scripts/modules/amazon/src/loadBalancer/TargetGroupDetails.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Overridable, IOverridableProps } from '@spinnaker/core';
+
+export interface ITargetGroupDetailsProps extends IOverridableProps {
+}
+
+@Overridable('loadBalancer.targetGroupDetails')
+export class TargetGroupDetails extends React.Component<ITargetGroupDetailsProps> {
+  public render() {
+    return (
+      <h3>Target Group Details</h3>
```

Comment:
Default content (for now).

As @jrsquared suggested, we might want to throw an error for cloud provider overrides if there is no override configured for the cloud provider.